tests/test_fetch.py

```diff
@@ -334,6 +334,10 @@ def test_product_type():
     producttype = era5._product_type()
     assert producttype is None
 
+    era5.ensemble = True
+    with pytest.raises(ValueError):
```

Comment:
See comment above. If you choose `assert` in `cli.py` instead of `ValueError`, don't forget to change this to `AssertionError` (I know I would) ;-)